recipes/proj/all/conanfile.py

```diff
@@ -109,6 +110,7 @@ def generate(self):
             # Workaround for: https://github.com/conan-io/conan/issues/13560
             libdirs_host = [l for dependency in self.dependencies.host.values() for l in dependency.cpp_info.aggregated_components().libdirs]
             tc.variables["CMAKE_BUILD_RPATH"] = ";".join(libdirs_host)
+        tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
```

Comment:
I'm curious, why is it needed?

Reply:
Afaik Valgur has been adding these in case upstream ever adds an option before project etc that can't be then controlled from Conan in old Cmake versions

Reply:
It doesn't work like this. Policy doesn't depend on CMake version you are using but on CMake version defined in cmake_minimum_required().
This line is very likely useless.

Reply:
Looking at conancenter history, I see this policy injected more & more quite randomly without rational. It's quite misleading.

Reply:
I agree with @SpaceIm here.